modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppRestSdkClientCodegen.java

```diff
@@ -208,7 +208,8 @@ public String toModelImport(String name) {
         if (importMapping.containsKey(name)) {
             return importMapping.get(name);
         } else {
-            return "#include \"" + name + ".h\"";
+            // Use relative imports unless this model is explicitly import-mapped.
+            return "#include \"../model/" + name + ".h\"";
```

Comment:
why not use /model/ + name + .h

Reply:
This would require the user of the generated code to specify an extra include path just to compile it. Using a proper relative path ensures that the library can be compiled without the need for extra path manipulation.